src/main/webapp/app/overview/course-exercises/course-exercises.component.ts

```diff
@@ -60,15 +84,68 @@ export class CourseExercisesComponent implements OnInit, OnDestroy {
         this.paramSubscription.unsubscribe();
     }
 
-    public groupExercises(selectedOrder: number): void {
+    get filterType() {
+        return ExerciseFilter;
```

Comment:
should this be moved as a `readonly` property to the same place as `ExerciseSortingOrder`?

Reply:
True, I will refactor it